ansible/fatimage.yml

```diff
@@ -120,28 +120,34 @@
       when: "'openhpc' in group_names"
 
     # - import_playbook: portal.yml
-    - name: Open Ondemand server (packages)
+    - name: Open OnDemand server (packages)
       include_role:
         name: osc.ood
         tasks_from: install-package.yml
         vars_from: "Rocky/{{ ansible_distribution_major_version }}.yml"
       when: "'openondemand' in group_names"
     # # FUTURE: install-apps.yml - this is git clones
 
-    - name: Open Ondemand server (apps)
+    - name: Open OnDemand server (apps)
       include_role:
         name: osc.ood
         tasks_from: install-apps.yml
         vars_from: "Rocky/{{ ansible_distribution_major_version }}.yml"
       when: "'openondemand' in group_names"
 
-    - name: Open Ondemand remote desktop
+    - name: Open OnDemand certbot
+      import_role:
+        name: openondemand
+        tasks_from: certbot.yml
```

Comment:
This seems to actually generate the LE cert? But this task is part of image build - how can that work? I'd have expected to only install the certbot package during image build, then run it during site.yml - what am I missing?